docs/openapi/open-api.yml

```diff
@@ -69,3 +138,68 @@ components:
         - mSettings
         - mStandings
         - mTeam
+        - players_wl
+        - proTeamSchedules_wl
+        - mDraftDetail
+    
+    TransactionPayload:
+      description: Payload for POST request body to change a lineup
+      type: "object"
+      required:
+        - isLeagueManager
+        - teamId
+        - type
+        - memberId
+        - scoringPeriodId
+        - executionType
+        - items
+      properties:
+        isLeagueManager: 
+          description: not sure if this can be boolean and not sure what happens if its true
+          type: string
+          enum:
+            - "false"
+            - "true" 
+        teamId:
+          description: the team id for the roster to be updated
+          type: integer
+        type: 
+          description: TBD, maybe LM as true can make retroactive changes, make add/drops
+          enum:
+            - "FUTURE_ROSTER" 
+        memberId: 
+          description: SWID
+          type: string
+        scoringPeriodId: 
+          description: the scoring period (day) for the lineup change
+          type: integer
+        executionType: 
+          description: not sure
+          type: string
+          enum: 
+            - "EXECUTE"
+        items: 
+          description: list of lineup instructions
+          type: array
+          items:
+            $ref: '#/components/schemas/Instruction'
+
+    Instruction:
```

Comment:
"LineupChangeInstruction"

Reply:
same as above, this might be used for add/drop transactions as well

Reply:
Then maybe "TransacrionInstruction" if we are going to be generic.